doorstop/core/test/files/index.html

```diff
@@ -130,8 +130,8 @@
 <p>
 <ul>
 <li> <a href="index2.html">index2</a> </li>
-<li> <a href="published.html">published</a> </li>
 <li> <a href="published2.html">published2</a> </li>
+<li> <a href="published.html">published</a> </li>
```

Comment:
I've noticed that this file keeps changing on your PRs, so I assume the issue is OS-specific.

Do you think you could add a fix that ensures these filenames are sorted according to Python sort order?